opennms-webapp/src/main/webapp/includes/grafana-box.jsp

```diff
@@ -0,0 +1,132 @@
+<%@taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>
+
+<%
+    String grafanaApiKey = System.getProperty("org.opennms.grafanaBox.apiKey", "");
+    String grafanaProtocol = System.getProperty("org.opennms.grafanaBox.protocol", "http");
+    String grafanaHostname = System.getProperty("org.opennms.grafanaBox.hostname", "localhost");
+    String grafanaTag = System.getProperty("org.opennms.grafanaBox.tag", "");
+    int grafanaPort = Integer.parseInt(System.getProperty("org.opennms.grafanaBox.port", "3000"));
+
+    String responseString = null;
+
+    if (!"".equals(grafanaApiKey)
+            && !"".equals(grafanaHostname)
+            && !"".equals(grafanaProtocol)
+            && ("http".equals(grafanaProtocol) || "https".equals(grafanaProtocol))) {
+        /**
+         * Creating the Http client instance...
+         */
+        DefaultHttpClient httpclient = new DefaultHttpClient();
+
+        HttpHost httpHost = new HttpHost(grafanaHostname, grafanaPort, grafanaProtocol);
+
+        /**
+         * ...using the search API url.
+         */
+        HttpGet httpGet = new HttpGet("/api/search/");
+        httpGet.setHeader("Authorization", "Bearer " + grafanaApiKey);
+
+        /**
+         * Invoke the request to the Grafana host.
+         */
+        HttpResponse httpResponse = httpclient.execute(httpHost, httpGet);
+
+        /**
+         * Convert the complete response into a string...
+         */
+        responseString = IOUtils.toString(httpResponse.getEntity().getContent(), "UTF-8");
+    }
+%>
+
+<div id="heatmap-box" class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title">Grafana Dashboards</h3>
+    </div>
+    <div class="panel-body">
+        <script type="text/javascript" src="/opennms/js/jquery/jquery-1.8.2.min.js"></script>
+        <script type="text/javascript" src="/opennms/js/jquery/ui/jquery-ui-1.8.2.custom.js"></script>
+        <%
+            if (responseString != null) {
+        %>
+        <script type="text/javascript">
+            var grafanaTag = '<%=grafanaTag%>';
+            var obj = <%=responseString%>;
+
+            jQuery.each(obj['dashboards'], function (i, val) {
+                var showDashboard = true;
+
+                if (grafanaTag != '') {
+                    showDashboard = false;
+                    for (var index in val['tags']) {
+                        console.log(val['tags'][index]);
+                        if (grafanaTag == val['tags'][index]) {
+                            showDashboard = true;
+                            break;
+                        }
+                    }
+                }
+
+                if (showDashboard) {
+                    document.write('<a href="<%=grafanaProtocol%>://<%=grafanaHostname%>:<%=grafanaPort%>/dashboard/db/' + val['slug'] + '"><span class="glyphicon glyphicon-signal" aria-hidden="true"></span>&nbsp;' + val['title'] + "</a><br />");
```

Comment:
The list of URLs should be rendered as a [list](http://getbootstrap.com/css/#unstyled) instead of using <br />. The former makes it easier to style or theme.